lib/init.coffee

```diff
@@ -464,3 +468,15 @@ module.exports =
         else
           # The file is to be ignored, we therefore return an empty set of warning.
           return []
+
+  resolvePath: (targetPath) ->
+    editor = atom.workspace.getActivePaneItem()
+    filepath = editor?.buffer?.file?.path
+
+    if not filepath
+      return targetPath
+
+    [projectPath, ...] = atom.project.relativizePath(filepath)
+    projectName = path.parse(projectPath).name
```

Comment:
`projectPath` can be `null` if the requested `filepath` is not part of an open Atom project (the file was opened on its own). You should be checking for that before attempting to `path.parse` it.